docs/application/toc_all.md

```diff
@@ -670,6 +670,7 @@
 ####### [Datetime](/application/native/guides/ui/efl/wearable/component-datetime.md)
 ####### [Entry](/application/native/guides/ui/efl/wearable/component-entry.md)
 ####### [Genlist](/application/native/guides/ui/efl/wearable/component-genlist.md)
+####### [Gengrid](/application/native/guides/ui/efl/wearable/component-gengrid.md)
```

Comment:
Would you please follow the alphabetical order?

```suggestion
####### [Gengrid](/application/native/guides/ui/efl/wearable/component-gengrid.md)
####### [Genlist](/application/native/guides/ui/efl/wearable/component-genlist.md)
```

Reply:
Thanks for the update. However, would you please check this comment again? It seems to be missed. 
Still "Genlist" comes before "Gengrid".